versions.yaml

```diff
@@ -0,0 +1,34 @@
+# This file defines all the Istio versions supported by this operator.
```

Comment:
Perhaps we could have two main sections: `versions` and `aliases` where in `aliases ` we could say things like `stable` or `latest` or `stable-1.19`, etc... just thinking out loud...

Reply:
Not sure why we'd need this. If the need arises, we can make these changes then. Let's try to add just the stuff that we're sure we need. Otherwise we're potentially wasting time.

Reply:
Yeah, I was thinking if users could just reference `stable` in istio.spec.version and that could be translated to an actual version such as `v1.19.5`. But that would probably mean automatic upgrades...

Reply:
For now, I think we'll even remove the `latest` version. At least from the product build, since it's not compatible with upgrades. Likewise, I don't think we'll want to add versions like `v1.19` or `stable` or anything that isn't one specific version.

We may add versions like this to the community operator, but with a warning that the upgrade behavior is undefined.